CONTRIBUTING.rst

```diff
@@ -0,0 +1,230 @@
+Contributing
```

Comment:
Is this a symbolic link to the contributor guide?  Are we going to maintain two versions of the same document?

Reply:
Where do you see a symbolic link?